src/app/components/FormElements/DaysOfWeekChecklist.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import * as EnumUtils from '../../infrastructure/EnumUtils';
+import { Checkbox } from 'react-bootstrap';
+import { default as FormFieldWrapper, FormFieldWrapperProps } from './FormFieldWrapper';
+import { DaysOfWeek } from '../../api';
+
+
+
+export default class DaysOfWeekChecklist extends React.PureComponent<FormFieldWrapperProps>{
+    render() {
+        const { input: { onChange } } = this.props;
+        return (
+            <FormFieldWrapper {...this.props}>
+                <br/>{EnumUtils.getEnumKeysAsStrings(DaysOfWeek).filter( k => k != 'Weekdays' && k != 'Everyday').map((k) =>
```

Comment:
Instead of using `<br/>` and `{'  '}` to create space.  Sometimes its easier and more consistently rendered if you use styles.  To do what you're trying to do here it would look like:
```tsx
  <div style={{marginTop:10}}> {....enumstuff.map(k=><Checkbox style={{paddingLeft:5}} ...>k</Checkbox>}
```